admin documentation (#198)
maykinmedia/objects-api: docs/admin/objecttype.rst

.. _admin_objecttype:

============
Object types
============

Creating and changing object types are common tasks for the Objecttypes API.
:ref:`api_usage` describes how to do it with HTTP requests. But you can also

> use the admin interface to create, publish and update object types.

Reviewer: ```suggestion
use the Objecttypes admin to create, publish and update object types.
```

          "examples": [
            "Laanboom"
          ],
          "enum": [
            "Laanboom",
            "Boomweide",
            "Solitaire boom"
          ]
        },
        "boomhoogteactueel": {
          "$id": "#/properties/boomhoogteactueel",
          "type": "integer",
          "title": "BoomhoogteActueel",
          "description": "Hoogte van de boom in meters.\nEenheid: m"
        },
        "leeftijd": {
          "$id": "#/properties/leeftijd",
          "type": "integer",
          "title": "Leeftijd",
          "description": "Leeftijd van het beheerobject in jaren.\nEenheid: Aantal",
          "examples": []
        },
        "meerstammig": {
          "$id": "#/properties/meerstammig",
          "type": "boolean",
          "title": "Meerstammig",
          "description": "Aanduiding voor meerstammigheid bij een Boom"
        },
        "type": {
          "$id": "#/properties/type",
          "type": "string",
          "title": "Type",
          "description": "Typering van het beheerobject."
        }
      },
      "required": ["boomhoogteactueel", "leeftijd"]
    }

> In the admin page of the Objects API click on the "add" button for the "Object types"
> resource.

Reviewer: ```suggestion
In the Objecttypes admin dashboard click on "Objecttypes" and click on the "Add" button.
```

Oops Objects API :p

Author: thanks for catching it :)
